src/lib/cache.test.ts

```diff
@@ -63,6 +63,9 @@ describe('cache', () => {
 
         expect(fn).toHaveBeenCalledTimes(1);
 
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const x = await Promise.resolve(''); // necessary for running test locally, for some reason
+
```

Comment:
Gosh - the only change to the cache was the fixing the retry interval wasn't it?

Reply:
I think this is unrelated to this change. The test was failing locally but not in teamcity!

Reply:
I've just done a little debugging. It's something weird going on with that finally block. Without the await, it doesn't run until after the assertion has already happened. No idea why this isn't the case for teamcity though

Reply:
interesting!

Reply:
oh I've got it!

Reply:
Instead of 
```js
        await expect(fetchData()).rejects;
```

we want something like

```js
        await expect(fetchData()).rejects.toEqual(
            new Error('Failed to make initial request for test4: Error: ERROR'),
        );
```

Reply:
I think we're not actually awaiting on that promise at the moment

Reply:
That now passes locally without `await Promise.resolve(' ')`

Reply:
Great spot! That was not obvious...